docs/src/main/paradox/docs/delta/api/schemas-api.md

```diff
@@ -334,7 +337,7 @@ GET /v1/schemas?from={from}
   (containing) the provided string
 - `{sort}`: String - can be used to sort schemas based on a payloads' field. This parameter can appear multiple times 
   to enable sorting by multiple fields. The default is done by `_createdBy` and `@id`.
-
+- `{aggregations}`: Boolean - if `true` then the response will only contain aggregations of the `@type` and `_project` fields; defaults to `false`. See @ref:[Aggregations](resources-api.md#aggregations).
```

Comment:
It's possible that `aggregate` makes more sense since it's a command?

Reply:
We already had this discussion when we introduced aggregation on generic resources and this was the result. One of the reasons was there are already aggregate views around which are a different thing